frontend/src/scenes/onboarding/sdks/session-replay/SessionReplaySDKInstructions.tsx

```diff
@@ -30,4 +33,7 @@ export const SessionReplaySDKInstructions: SDKInstructionsMap = {
     [SDKKey.SVELTE]: SvelteInstructions,
     [SDKKey.VUE_JS]: VueInstructions,
     [SDKKey.WEBFLOW]: WebflowInstructions,
+    [SDKKey.IOS]: iOSInstructions,
+    [SDKKey.ANDROID]: AndroidInstructions,
```

Comment:
we're adding these two in `Onboarding.tsx`

```
    const sdkInstructionMap = SessionReplaySDKInstructions
    if (hasMobileOnBoarding) {
        sdkInstructionMap[SDKKey.ANDROID] = AndroidInstructions
        sdkInstructionMap[SDKKey.IOS] = iOSInstructions
    }
    ```
    
    which is controlled in there by
    
    ```
const { featureFlags } = useValues(featureFlagLogic)
    const hasMobileOnBoarding = !!featureFlags[FEATURE_FLAGS.SESSION_REPLAY_MOBILE_ONBOARDING]
```

I guess we can remove that flag and check here too?

Reply:
yep, been out for 100% of users for months

https://us.posthog.com/project/2/feature_flags/45369

Reply:
true that, good point, pushed a PR

Reply:
mind reviewing https://github.com/PostHog/posthog/pull/25237 as well? both will probably conflict